ch_pipeline/processing/daily.py

```diff
@@ -622,7 +622,75 @@
         output_name: "ringmap_intercyl_hpf_{{tag}}.zarr.zip"
         remove: true
 
+    # Take the variance over frequency for the XX pol of the delay-filtered
+    # ringmap
+    - type: draco.analysis.transform.Reduce
+      in: ringmap_int_hpf
+      out: ringmap_reduced_freq
+      params:
+        axes: ["freq"]
+        datasets: ["map"]
+        op: "var"
+        selections: {"pol":[0, 3]}
+        generate_weight: true
+        apply_weight: false
+
+    - type: draco.core.io.Truncate
+      in: ringmap_reduced_freq
+      out: ringmap_reduced_freq_trunc
+      params:
+        dataset:
+          map: No
+          weight: 1.0e-5
+
+    - type: draco.core.io.SaveZarrZip
+      in: ringmap_reduced_freq_trunc
+      out: ringmap_reduced_freq_trunc_handle
+      params:
+        compression:
+          map:
+            chunks: [1, 1, 32, 512, 512]
+          weight:
+            chunks: [1, 32, 512, 512]
+        save: true
+        output_name: "ringmap_intercyl_freq_var_{{tag}}.zarr.zip"
+        remove: true
+
+    - type: draco.analysis.transform.Reduce
+      in: ringmap_int_hpf
+      out: ringmap_reduced_el
+      params:
+          axes: ["el"]
+          datasets: ["map"]
+          op: "var"
+          selections: {"pol":[0, 3]}
+          generate_weight: true
+          apply_weight: false
```

Comment:
Making sure params agree with `Reduce` task:
```suggestion
          generate_mask: true
          apply_mask: false
```